articles/concepts/Glossary.md

```diff
@@ -12,31 +12,33 @@ uid: microsoft.quantum.glossary
+|<a id="epr"></a>EPR pair|A maximally entangled quantum state of two qubits. Also known as a [Bell state](xref:microsoft.quantum.glossary#bell-state)|
+|<a id="evolution"></a>Evolution|How a quantum state changes over time. For more information, see [Matrix exponentials](xref:microsoft.quantum.concepts.matrix-advanced#matrix-exponentials).|
+|<a id="function"></a>Function|A purely classical (non-quantum) routine in the Q# language. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|
+|<a id="global-phase"></a>Global phase|When two states are identical up to a multiple of a complex number $e^{i\phi}$, they are said to differ up to a global phase. Unlike local phases, global phases cannot be observed through any measurement. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|
+|<a id="measurment"></a>Measurement|Obtaining a classical bit from a qubit (or set of qubits). FOr more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit#measuring-a-qubit).|
+|<a id="mutable"></a>Mutable|A variable whose value may be changed after it is created.|
+|<a id="namespace"></a>Namespace|A label for a collection of related names (typically operations, functions, and types). For instance the namespace [Microsoft.Quantum.Preparation](xref:microsoft.quantum.preparation) labels all of the symbols defined in the standard library that help with preparing initial states.|
+|<a id="operation"></a>Operation|The basic unit of quantum execution in Q#. It is roughly equivalent to a function in C, C++ or Python, or a static method in C# or Java. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|
+|<a id="operator-application"></a>Operator application|Performing a quantum operation. This typically applies a unitary matrix to the current quantum   state vector.|
+|<a id="oracle"></a>Oracle|A subroutine that provides data-dependent information to a quantum algorithm at runtime. Typically, the goal is to provide a superposition of outputs corresponding to inputs that are in superposition. For more information, see [Oracles](xref:microsoft.quantum.libraries.data-structures#oracles).|
+|<a id="partial-app"></a>Partial application|Calling a function or operation without all the required parameters. The returns a new callable that only needs the missing parameters supplied during a future application. For more information, see [Partial application](xref:microsoft.quantum.libraries.data-structures#partial-application).|
+|<a id="pauli"></a>Pauli operators|The `X`, `Y` and `Z` quantum gates. For more information, see [Single-qubit operations](xref:microsoft.quantum.concepts.qubit#single-qubit-operations).|
```

Comment:
At a least, I'd suggest "operations" instead of "gates," as a "gate" can be a colloquialism for either an operation or an operator, depending on context. That said, the original definition here was likely wrong in that it conflated 𝑋, `PauliX` and `Microsoft.Quantum.Intrinsic.X` with each other.
```suggestion
|<a id="pauli"></a>Pauli operators|The `X`, `Y` and `Z` quantum operations. For more information, see [Single-qubit operations](xref:microsoft.quantum.concepts.qubit#single-qubit-operations).|
```